### PR TITLE
Some minor updates

### PR DESCRIPTION
**Pull request by VSEphpbb**

### CODE REVIEW

File: event/listener.php [hunk 1]
```diff
@@ -115,7 +115,7 @@ public function show_page_links()
 
 			// Assign any available custom icons for the current link in the user's style
 			$custom_icon = '';
-			foreach ($pages_icons as $icon_path => $ext_name)
+			foreach (array_keys($pages_icons) as $icon_path)
```
Comment (michaelcullum) [inline event/listener.php]: Why?

Comment (VSEphpbb) [inline event/listener.php]: We only need the keys. $ext_name is never used.

Comment (michaelcullum) [inline event/listener.php]: This is probably going to have a tiny perf hit though, I'm not sure it serves any purpose.

**Comment by VSEphpbb** [comment]
😦

**Pull request merged**